DuckDuckGo/AppDelegate.swift

```diff
@@ -435,7 +435,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
 #endif
             SubscriptionPurchaseEnvironment.current = .appStore
-            await AccountManager().checkSubscriptionState()
+            await AccountManager().refreshSubscriptionAndEntitlements()
```

Comment:
What about moving this from didFinishLaunchingWithOptions to applicationDidBecomeActive so we can refresh the cache every time the app comes back from the background? didFinishLaunchingWithOptions could not happen for days

Reply:
+1 to having this invoked more often.

Reply:
There is already call below in `applicationWillEnterForeground` but will move it to `applicationDidBecomeActive`